### PR TITLE
Implementing brand container, nav bar, and links.

### PR DESCRIPTION
**Pull request by chykB**
I deleted the About page and renamed the contact page to the Blog page, we also made the below changes as a team.
1. Please change the title of the home page to “Welcome to Algorithmia SE 🌍”
2. Upload the website logo for the home page. It is in the [images/website-logo/](https://github.com/ALX-SE-Algorithmia/ALX-SE-Algorithmia.github.io/tree/main/images/website-logos) directory of the repository.
3. Link the [Bentham font](https://fonts.google.com/specimen/Bentham?query=Bentham&sort=alpha) to the home page.
4. Try linking the CDNs (Iconscout CDN and Google Font CDN) if possible, to implement the icons on the website.

### CODE REVIEW

**Comment by dohoudaniel** [comment]
Hi @chykB, @dohoudaniel here.
Thank you for making **the very first** contribution to this project.
However, there are some changes that need to be made:
1. Please change the title of the home page to “Welcome to Algorithmia SE 🌍”
2. Upload the website logo for the home page. It is in the <a href="https://github.com/ALX-SE-Algorithmia/ALX-SE-Algorithmia.github.io/tree/main/images/website-logos" target="_blank">images/website-logo/</a> directory of the repository.
3. Link the <a href="https://fonts.google.com/specimen/Bentham?query=Bentham&sort=alpha">Bentham font</a> to the home page.
4. Try linking the CDNs (Iconscout CDN and Google Font CDN) if possible, to implement the icons on the website.

However, please note that you are not to work on this alone.
You and the assigned people in this <a href="https://github.com/ALX-SE-Algorithmia/ALX-SE-Algorithmia.github.io/issues/10">open issue</a> will be working on it.
Thank you.

**Comment by dohoudaniel** [comment]
Hi @chykB.
Thank you for making edits to your PR.
However, there are some new updates and edits made.
The About Page and Contact Pages are no longer being implemented as seperate HTML pages.
The pages we now have are:
- Home Page
- Blog Page
- Events Page
- Projects Page.

Kindly see if you can make these edits in your PR.
Thank you for understanding.

File: about.html
Comment (dohoudaniel) [inline about.html:1]: Hi @chykB.
Please kindly delete this file.

File: contact.html
Comment (dohoudaniel) [inline contact.html:1]: Hi @chykB.
Please edit this file, and rename it to blogs.html
Thank you.

File: events.html
Comment (dohoudaniel) [inline events.html:1]: You could also edit this file as appropriate.

File: index.html
Comment (dohoudaniel) [inline index.html:1]: You could also edit this file as appropriate.

File: contact.html
Comment (dohoudaniel) [reply contact.html:1]: You could also edit this file as appropriate.

File: projects.html
Comment (dohoudaniel) [inline projects.html:1]: You could also edit this file as appropriate.

File: about.html
Comment (chykB) [reply about.html:1]: changes has been done

File: contact.html
Comment (chykB) [reply contact.html:1]: Changes have been made.

File: blog.html
Comment (dohoudaniel) [inline blog.html:1]: @chykB Kindly rename this file to blogs.html

**Review by dohoudaniel — APPROVED** [review]

**Pull request merged**